README.md

```diff
@@ -47,42 +47,49 @@ You can find a full list of our demos and view the code for each of them in the
 Full documentation is available at [https://geofirestore.com](https://geofirestore.com). It mostly provides the same functionality as the Firestore library, in almost the same way as the Firestore library. Many questions can be addressed by looking at the [Firestore docs](https://firebase.google.com/docs/firestore/). However there are a few differences, and below is a little example of how to make a location based query.
 
 ```TypeScript
+// If you're using TypeScript
 import * as firebase from 'firebase/app';
 import 'firebase/firestore';
 
 // If you're using ES6+/imports/ESM syntax for imports you can do this:
 import { GeoCollectionReference, GeoFirestore, GeoQuery, GeoQuerySnapshot } from 'geofirestore';
+```
+
+```Javascript
+// If you're using JavaScript
+var admin = require("firebase-admin");
+var serviceAccount = require("./service-account-key.json");
 
-// If you're using CommonJS/require syntax for imports you can do this:
 const { GeoCollectionReference, GeoFirestore, GeoQuery, GeoQuerySnapshot } = require('geofirestore');
 
-// Initialize the Firebase SDK
-firebase.initializeApp({
-  // ...
+// Initialize the Firebase App
+admin.initializeApp({
```

Comment:
I get this empty for ppl to initialize their firebase app as the please. I'd ask you to revert back to `firebase` over `admin`.